infra/recipes/terraform/azure/sandbox/kubernetes-services.tf

```diff
@@ -0,0 +1,17 @@
+resource "azurerm_kubernetes_cluster" "sandbox" {
+    name = "sandbox-k8s"
+    resource_group_name = azurerm_resource_group.sandbox.name
+    location = azurerm_resource_group.sandbox.location
+    dns_prefix = "sandboxk8s"
+
+    default_node_pool {
+      name = "default"
+      node_count = var.node_count
```

Comment:
Do we need to update this variable in variables.tf to increase the node count?

Reply:
Yes I believe so. I just extracted it out as a variable so it is easily changeable.